index.html

```diff
@@ -96,59 +96,111 @@ <h3>Terminology</h3>
-		  
-		  <!-- here are the direct links
-		  <dl>
-			  <li><a href="https://github.com/w3c/i18n-activity/issues/111">111</a></li>
-			  <li><a href="https://github.com/w3c/i18n-activity/issues/110">110</a></li>
-			  <li><a href="https://github.com/w3c/i18n-activity/issues/109">109</a></li>
-			  <li><a href="https://github.com/w3c/i18n-activity/issues/108">108</a></li>
-			  <li><a href="https://github.com/w3c/i18n-activity/issues/107">107</a></li>
-			  <li><a href="https://github.com/w3c/i18n-activity/issues/106">106</a></li>
-			  <li><a href="https://github.com/w3c/i18n-activity/issues/105">105</a></li>
-			  <li><a href="https://github.com/w3c/i18n-activity/issues/104">104</a></li>
-			  <li><a href="https://github.com/w3c/i18n-activity/issues/103">103</a></li>
-		  </dl>
-		  -->
-      </div>
+      <h2>Searching Text in Natural Language Content</h2>
       
-      <p>Users of the Web often want to search documents for particular words or phrases within the natural language text of a given document. This is different from the sorts of programmatic matching needed by formal languages (such as markup languages such as [[HTML]]; style sheets [[CSS21]]; or data formats such as [[TURTLE]] or [[JSON-LD]]), and which are described by our document [[CHARMOD-NORM]]. </p>
+      <p>Users of the Web often want to search for specific text in a document or collection of documents without having to read line-by-line. Specifications sometimes seek to support this desire by exposing text searching in the Web platform.</p>
       
-      <p>There are different types of string searching. 
+      <p>There are different types of document searching. One type, called a <a>full text search</a>, is the sort of searching most often found in applications such as a search engine. This type of searching is complex, can be resource intensive, and often depends on processes outside the scope of a given search request.</p> 
     
-      <p>One limited form of full-text search&mdash;and the topic of this document&mdash;is sub-string matching. One familiar form of sub-string matching is the "find" feature of your browser. A sub-string match searches the body ("<a>corpus</a>") of a document with the user's input, seeking a match.</p>
+      <p>A more limited form of text search (and the topic of this document) is <q>sub-string matching</q>. One familiar form of sub-string matching is the <q><em>find</em></q> feature of browsers and other types of user-agent. In browsers, this functionality is often accessed via a key combination such as <kbd>Cmd+F</kbd> or <kbd>Ctrl+F</kbd>. Such a feature might be exposed on the Web via the API <code translate=no>window.find</code>, which is currently not fully standardized, or capabilities such as the proposed scroll-to-text-fragment.</p>
```

Comment:
```suggestion
      <p>A more limited form of text search (and the topic of this document) is <q>sub-string matching</q>. One familiar form of sub-string matching is the <q><em>find</em></q> feature of browsers and other types of user-agent. In desktop browsers, this functionality is often accessed via a key combination such as <kbd>Cmd+F</kbd> or <kbd>Ctrl+F</kbd>. Such a feature might be exposed on the Web via the API <code translate=no>window.find</code>, which is currently not fully standardized, or capabilities such as the proposed scroll-to-text-fragment [[SCROLL-TO-TEXT-FRAGMENT]].</p>
```

Reply:
Thanks. I did slight edits to this in my changes.